more_itertools/more.py

```diff
@@ -1004,3 +1004,53 @@ def divide(n, iterable):
         ret.append(iter(seq[start:stop]))
 
     return ret
+
+
+def always_iterable(obj):
+    """
+    Given an object, always return an iterable.
+
+    If the object is not already iterable, return a tuple containing containing
+    the object::
+
+        >>> always_iterable(1)
+        (1,)
+
+    If the object is ``None``, return an empty iterable::
```

Comment:
In my experience, empty for None is almost always what you want. I've never encountered a case where one would want None in a tuple. I'll dig up some examples.